FILE_TEMPLATE

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2022 Parity Technologies (UK) Ltd. (admin@parity.io)
+// Copyright (C) 2024 Parity Technologies (UK) Ltd. (admin@parity.io)
```

Comment:
dq; maybe we should ommit the copyright year, similar to polkadot-sdk?

Reply:
Ooh good thought! I'm hesitant to do that right now because the polkadot-sdk license is different and so I'd have to check whether it was true for the apache 2 license too that no year was needed. From a quick google I also gather that leaving the old year there doesn't really have any impact!